interface/client/src/App.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import 'antd/dist/antd.css';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+
+import * as store from './store';
+import { Layout } from 'antd';
+import { AppHeader } from './ui/AppHeader';
+import { AppMain } from './ui/AppMain';
+
+const { Header, Content, Footer } = Layout;
+
+interface AppContainerProps {
+    store?: store.Store;
+}
+
+function mapStateToProps(state: store.Store): any {
+    return {
+        store: state,
+    };
+}
+
+export class AppContainerImpl extends React.Component<AppContainerProps, any> {
+    private renderMainUI(): React.ReactNode {
+        return (<Layout className="mainLayout">
+        <Header>
+            <AppHeader />
+        </Header>
+        <Content className="mainContent">
+            <AppMain />
```

Comment:
Leave only this line `<AppMain />` and remove others

Reply:
Still actual. Leave only `<AppMain />`, remove Layout, Footer and other. 
Remove `Header, Content, Footer` consts as they will not be used